sources/core/Stride.Core.Mathematics/Vector2.cs

```diff
@@ -207,6 +207,24 @@ public float[] ToArray()
             return new float[] { X, Y };
         }
 
+        /// <summary>
+        /// Moves the first vector2 to the second one in a straight line.
+        /// </summary>
+        /// <param name="First">The first point.</param>
+        /// <param name="Second">The second point.</param>
+        /// <param name="MaxDistancePerCall">The rate at which the first point is going to move towards the second point.</param>
+        public static Vector2 Goto(Vector2 First, Vector2 Second, float MaxDistancePerCall)
```

Comment:
there could be a version with `ref/out` parameters to save copies. and the version without them will just call the other, like it is done with most math functions.